javascript/ql/test/ApiGraphs/accessors/index.js

```diff
@@ -0,0 +1,65 @@
+const foo = require('foo');
+
+foo({
+    myMethod(x) { /* use (parameter 0 (member myMethod (parameter 0 (member exports (module foo))))) */
```

Comment:
```suggestion
    myMethod(x) { /* use=moduleImport("foo").getMember("exports").getParameter(0).getMember("myMethod").getParameter(0) */
```

[I recently changed the testing syntax](https://github.com/github/codeql/pull/8578).  
All your tests need to be rewritten to the new syntax, because currently they don't test anything.